src/help-widget.ts

```diff
@@ -205,4 +209,27 @@ export class BBHelpHelpWidget {
   private isCollapsed() {
     return this.container.classList.contains(HELP_CLOSED_CLASS);
   }
+
+  private watchWindowWidth() {
+    window.addEventListener('resize', () => {
+      this.resizeContainer();
+    });
+  }
+
+  private resizeContainer() {
+    if (window.innerWidth < SCREEN_XS_MAX) {
```

Comment:
Two things here:
A) with the current logic, these classes are being added and removed over and over on resize, which causes some stuttering and lag.  We should create a variable or some kind of flag to tell if things have been set already or not. e.g
```
    if (this.isSetForMobile !== false) {
      this.isSetForMobile = false;
      this.container.classList.remove('bb-help-container-mobile');
      this.container.classList.remove('bb-help-closed-mobile');
      this.invoker.classList.remove('bb-help-invoker-hidden');
    }
```
and B) How do you feel about breaking the larger chunk into a separate method so it's easier to read than nested if / else statments.

```
  private handleMobileSizing() {
    if (this.isSetForMobile === true) {
      return;
    }

    this.isSetForMobile = true;

    if (!this.container.classList.contains('bb-help-closed')) {
      this.container.classList.remove('bb-help-closed-mobile');
      this.container.classList.add('bb-help-container-mobile');
    } else {
      this.container.classList.remove('bb-help-container-mobile');
      this.container.classList.add('bb-help-closed-mobile');
    }

    this.invoker.classList.add('bb-help-invoker-hidden');
  }
```